Information/Design.md

```diff
@@ -62,8 +62,25 @@
 
```

Comment:
There is not any kind of return value stated for **REQ 070** . Moreover, there is not any command name created for this **REQ 070**.  Even further, even if you state explicitly what the return value will be for this command and create a command name for this requirement.   You would then need to explain WHY you choice these options versus other potential options for a command name, and a return value. Additionally you should consider cases in which the user calls the command name without any parameters, or if they choose to call the command with wrong parameters or in correct values. (i.e. with and integer or floating point v.s. a string) ECT.

Reply:
I have updated the design.md with more details for each of the requirements that needed it.

Reply:
Ok, I like the adjustments. I would like to see more on documentation on how you approached designing these commands. Even further, how would you compare and contrast other design approaches.  These are expected aspects the Design.md file that the Professor expects from us.  So please don't get mad. I am simply looking out for all of us.

Reply:
I have added information to the file, I believe it should suit the changes you requested.

Reply:
Ok, looks good.